Mathlib/Tactic/LinearCombination.lean

```diff
@@ -87,17 +87,12 @@ partial def expandLinearCombo (ty : Expr) (stx : Syntax.Term) : TermElabM Expand
     | .const c₁, .const c₂ => .const <$> ``($c₁ * $c₂)
     | .proof p₁, .const c₂ => .proof <$> ``(pf_mul_c $p₁ $c₂)
     | .const c₁, .proof p₂ => .proof <$> ``(c_mul_pf $p₂ $c₁)
-    | .proof p₁, .proof p₂ => .proof <$> ``(mul_pf $p₁ $p₂)
-  | `($e⁻¹) => do
-    match ← expandLinearCombo ty e with
-    | .const c => .const <$> `($c⁻¹)
-    | .proof p => .proof <$> ``(inv_pf $p)
+    | .proof _, .proof _ => throwError "'linear_combination' supports only linear operations"
```

Comment:
Can you add a test for this?

Reply:
I have to say, debugging the example above I found this message not immensely helpful. Is it possible that the error message mentions the term it is upset about?

Reply:
I added tests, and I localized the error to the offending `*` or `/` symbol -- what do you think?